web-app/components/Layout.tsx

```diff
@@ -18,15 +20,9 @@ export default function Layout({ children }: { children: JSX.Element | JSX.Eleme
           referrerPolicy="no-referrer"
         />
       </Head>
-      <div>
-        <a href="/">NEAR University Certificate Browser</a>
-      </div>
+      <Header />
```

Comment:
@ozanisgor This file looks so much cleaner now that Header and Footer are separate files. :-)